keyboards/keychron/k3_pro/ansi/white/keymaps/vial/config.h

```diff
@@ -0,0 +1,14 @@
+/* SPDX-License-Identifier: GPL-2.0-or-later */
+
+#pragma once
+
+#define VIAL_KEYBOARD_UID \
+    {0x60, 0xC5, 0xB1, 0xE4, 0x61, 0x9D, 0xBE, 0x9D}
+
+#define VIAL_UNLOCK_COMBO_ROWS \
+    { 0, 3 }
+
+#define VIAL_UNLOCK_COMBO_COLS \
+    { 0, 13 }
+
+#define BACKLIGHT_BREATHING
```

Comment:
```suggestion

```
Already exists as a lighting effect for [LED Matrix](https://github.com/adophoxia/vial-qmk/blob/vial-keychron-bluetooth-playground/keyboards/keychron/k3_pro/ansi/white/config.h#L46).